tests/test_jointcal_cfht.py

```diff
@@ -34,27 +22,40 @@ def setup_module(module):
 
 class JointcalTestCFHT(jointcalTestBase.JointcalTestBase, lsst.utils.tests.TestCase):
 
+    @classmethod
+    def setUpClass(cls):
+        try:
+            cls.data_dir = lsst.utils.getPackageDir('testdata_jointcal')
+            os.environ['ASTROMETRY_NET_DATA_DIR'] = os.path.join(cls.data_dir, 'cfht_and_index')
+        except lsst.pex.exceptions.NotFoundError:
+            cls.data_dir = None
```

Comment:
Nothing seems to be making the tests skip if this data dir is not set. The other tests raise the skip error.